app/styles/charts.less

```diff
@@ -0,0 +1,7 @@
+svg text {
+  font: 16px BlenderPro;
```

Comment:
I think the design does not use the Blender font any more.
So this is the last place where the font is used => remove font files.